nexus/README.md

```diff
@@ -0,0 +1,32 @@
+# Nexus
+
+Action to interact with Nexus.
+
+## Action usage
+
+### Get latest version of a package
+
+```yaml
+- uses: LedgerHQ/actions/nexus/latest-version
+  with:
+    feature-name: my-awesome-feature
+  env:
+    GREEN_NEXUS_HOST: <URL>
+    GREEN_NEXUS_USER: <username>
+    GREEN_NEXUS_PASSWORD: <password>
```

Comment:
Can you use the default secret names here? Like `${{ secrets.GREEN_NEXUS_HOST }}` instead of `<URL>`.

It will be easier for future use.

Reply:
I'm sorry I don't get what you mean